src/content/blog/idx-official-online-editor-partner.mdx

```diff
@@ -0,0 +1,58 @@
+
+IDX will be sponsoring $10,000 each month towards the ongoing open source maintenance and development of Astro. The new sponsorship funding will be invested directly back into our open source community and long-term maintenance, benefiting everyone.
+
+<BlogContentImage src={projectIDXSpace} alt="Dark graphic depicting outer space and the Project IDX logo." />
+
+## Commitment to Open Source
+
+Google has been committed to open-source software over the last two decades. 
+
+One of the most well-known examples is open-sourcing Chromium. This resulted in improved security, better developer tools, and a focus on web standards. Lesser known may be Google's internal culture. Google employees are incentivized to contribute to open source with the [Open Source Peer Bonus](https://opensource.google/documentation/reference/growing/peer-bonus). Google also also created [Google Summer of Code](https://summerofcode.withgoogle.com/) (GSoc). GSoc is a global online program that encourages new contributors to open source projects.  
+
+David East of IDX shares their thoughts on what this partnership means for both teams:
+
+<Blockquote>
+	<Fragment slot="quote" >
+		I've been a huge Astro fan since the early alpha days, and it's been awesome to see it grow into such a powerful framework. We chose it for idx.dev because we knew it would give us the speed and flexibility we needed, and now, partnering with the Astro team is a fantastic opportunity to give back to that community. Together, we're all about helping developers build the kind of fast, super dynamic, and truly accessible websites that represent the future of the web.
+	</Fragment>
+	<Fragment slot="caption">
+		— [David East](https://x.com/_davideast?lang=en), Lead DevRel at IDX
+	</Fragment>
+</Blockquote>
+
+The last time we partnered with David was to write a case study about Firebase cutting publishing time by building their blog with Astro. We're excited to continue our strong partnership with David, a long-time supporter of Astro that will be leading the partnership from the IDX side.
+
+<BlogContentImage src={logoGoogleOSS} alt="Light graphic depicting Google's OSS logo." />
+
+## Astro ❤️ Google IDX
+
+Astro is honored and excited to have IDX as our new official online editor partner. 
+
+To kick off our partnership, we have a refreshed experience on [astro.new](http://astro.new) so you can start your next Astro project even faster. Browse previews of Astro’s official starter templates and then launch in your browser with one click — or copy the CLI command and run `create astro` locally.
+
+It takes a village for an open-source project to thrive. We are grateful for the support of David East and the entire IDX team for their help and evangelism of open-source software. It's because of support from our partners and active community members that we are able to receive ongoing maintenance and keep developing Astro in an open, financially sustainable way. 
```

Comment:
nit
```suggestion
It takes a village for an open-source project to thrive. We are grateful for the support of David East and the entire IDX team for their help and evangelism of open-source software. It's because of support from our partners and active community members that we can receive ongoing maintenance and keep developing Astro in an open, financially sustainable way. 
```